platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,10 +1,19 @@
 #import "MGLSource.h"
+#import "MGLTypes.h"
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface MGLRasterSource : MGLSource
 
 @property (nonatomic, readonly, copy) NSURL *URL;
 @property (nonatomic, readonly, assign) CGFloat tileSize;
```

Comment:
`-mbglSource` casts this value to `uint16_t`, so this property shouldn’t be a floating-point number. (Or is the internal tile size in pixels rather than points?)